src/node_support/node_request_handler.ts

```diff
@@ -91,10 +93,8 @@ export class NodeBasedHandler extends AuthorizationRequestHandler {
       response.end('Close your browser to continue');
     };
 
-    this.authorizationPromise = new Promise<AuthorizationRequestResponse>((resolve, reject) => {
-      emitter.once(ServerEventsEmitter.ON_UNABLE_TO_START, () => {
-        reject(`Unable to create HTTP server at port ${this.httpServerPort}`);
-      });
+    this.authorizationPromise = new Promise<AuthorizationRequestResponse|null>((resolve) => {
+      emitter.once(ServerEventsEmitter.ON_UNABLE_TO_START, () => resolve(null));
```

Comment:
If the server could not be started the authorizationPromise is resolved with `null` to avoid unhandled Promise rejection. This can be safely done as we never would get a valid response anyway.

Reply:
I think we may want to `reject` this instead to be more consistent.

Reply:
Partially i agree. In the current state of the code base it should reject. Will change it in the PR. Please see my [comment](https://github.com/openid/AppAuth-JS/issues/135#issuecomment-565757383), why i think that it in fact should resolve with `null`.

Reply:
Should be resolved with 67e115d